scripts/schema-manager/setup.ts

```diff
@@ -0,0 +1,40 @@
+import { Tigris, TigrisClientConfig } from '@tigrisdata/core';
```

Comment:
if we do the above ^ then we don't need probably setup.ts? As once our client detects the model directory it can create the database and collection.

Reply:
```
async function main() {
  const tigrisClient = new Tigris();
  tigrisClient.initializeModels(); // basically a single function call to out client to setup everything
}
```

Reply:
We will still need it but a very simplified version of it.